src/Microsoft.Azure.CosmosRepository/Repositories/InMemoryRepository.cs

```diff
@@ -447,5 +448,26 @@ public async ValueTask<int> CountAsync(CancellationToken cancellationToken = def
 
         private void MismatchedEtags() =>
             throw new CosmosException(string.Empty, HttpStatusCode.PreconditionFailed, 0, string.Empty, 0);
+
+        public ValueTask UpdateAsBatchAsync(
+            IEnumerable<TItem> items,
+            CancellationToken cancellationToken = default)
+        {
+            throw new NotImplementedException();
+        }
+
+        public ValueTask CreateAsBatchAsync(
+            IEnumerable<TItem> items,
+            CancellationToken cancellationToken = default)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
```suggestion
        public async ValueTask CreateAsBatchAsync(
            IEnumerable<TItem> items,
            CancellationToken cancellationToken = default)
        {
            foreach (var item in items)
            {
                await CreateAsync(item);
            }
        }
```

Reply:
This will do for the time being, only issue is it's not a batch, I planned to come back to this but I think I agree this works for now.

The reason I say it's not a batch is just that if one etag fails, for example, some will persist, but not a huge issue I don't think.